task-list/app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Work on this:</h1>
+  <h3>Task </h3>
+    <p><%= @task.name%></p>
```

Comment:
These `p` tags are not nested inside the `h3` tags so they do not need to be indented.